notebooks/spark-compat.ipynb

```diff
@@ -130,8 +130,7 @@
       "inputCol: input column name. (undefined)\n",
       "inputCols: input column names. (undefined)\n",
       "k: the number of principal components (undefined)\n",
-      "num_workers: (cuML) number of Spark cuML workers, where each cuML worker corresponds to one Spark task. (default: 1)\n",
-      "outputCol: output column name. (default: PCA_fb9b083f9099__output)\n",
+      "outputCol: output column name. (default: PCA_5679f4201a43__output)\n",
```

Comment:
Looks like a lot of changes to notebook.  Just from rerunning?

Reply:
Yes, just re-ran that one, since it had the old num_workers output, and I merged the two linear regression sections.